src/arrayMethodSort.js

```diff
@@ -4,9 +4,41 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareAsStrings) {
+    let count;
+
+    do {
+      count = 0;
+
+      for (let i = 0; i < this.length; i++) {
+        const prev = this[i];
+        const current = this[i + 1];
+
+        if (compareFunction(prev, current) > 0) {
+          count++;
+          this[i] = current;
+          this[i + 1] = prev;
+        }
+      }
+    } while (count > 0);
+
+    return this;
   };
 }
 
+const compareAsStrings = function(a, b) {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  }
+
+  if (stringA === stringB) {
+    return 0;
+  }
+
+  return 1;
```

Comment:
you only need to return `1` if value is bigger
```suggestion
  return -1;
```